### ISSUE TITLE
User cant upload picture when he is trying to create Donne

### ISSUE THREAD

**Issue by ghost** [Goodler/code#165]
Steps:
 Log in
go to menu
tap Create Donee
enter all info and attach picture
tap create

Bug: no picture in Donee, even though it says loaded 100%
Bug: can we create a preview for picture?

**Comment by DmitrysIF** [comment]
А вы дождались загрузки картинок до 100%?

**Comment by ghost** [comment]
yeap. i saw 100%
Let me try again

**Comment by ghost** [comment]
@DmitrysIF this is what i see
File size 4MB load only till 87 % ( its been like that for 20 min)
Smaller sizes are better
DO we have any restrictions on size? 
![photo](https://cloud.githubusercontent.com/assets/7193843/2855299/0caece0a-d15b-11e3-8c4d-d9be86a08eef.png)

**Comment by DmitrysIF** [comment]
Я проверю.

**Comment by DmitrysIF** [comment]
@TShket  Donee создаются с картинками, не вижу проблемы. По поводу возникающей проблемы загрузки больших картинок - пока исследуется.
